src/app/(auth)/signup/page.tsx

```diff
@@ -105,34 +112,33 @@ export default function SignUp() {
             <BigButton type="button" onClick={handleSignUp}>
               Sign Up
             </BigButton>
-            <H4Centered>
+            <PCentered>
               Have an account already?{' '}
-              <LinkColored $color={COLORS.greyDark} href="/login">
+              <LinkColored $color={COLORS.blueMid} href="/login">
                 Log In
               </LinkColored>
-            </H4Centered>
+            </PCentered>
           </SpacerDiv>
         </>
       )}
       {emailSentCount > 0 && (
-        <SpacerDiv>
-          <H2>An email verification link has been sent.</H2>
-          <H4 $color={COLORS.greyDark}>
+        <SpacerDiv $align="center">
+          <Icon type="email" />
+          <AuthSubHeading>Verification email sent!</AuthSubHeading>
+          <H4Centered $color={COLORS.greyDark}>
             This link will direct you to the next step. If you didn’t receive an
             email, please click Resend Email.
-          </H4>
-          <HorizontalDiv>
-            <Button
-              $primaryColor={COLORS.blueMid}
-              $secondaryColor={COLORS.blueDark}
-              onClick={handleResendEmail}
-            >
-              <H4 $color="white">Resend Email</H4>
-            </Button>
-            {emailSentCount > 1 && (
-              <P $color={COLORS.greyMid}>Email has been resent!</P>
-            )}
-          </HorizontalDiv>
+          </H4Centered>
+          <Button
+            $primaryColor={COLORS.blueMid}
+            $secondaryColor={COLORS.blueDark}
+            onClick={handleResendEmail}
+          >
+            <H4 $color="white">Resend Email</H4>
+          </Button>
```

Comment:
Could we update the button to match the designs?

**Screenshot from your branch:**
<img width="678" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/adec13f5-73b5-4a12-b1b2-899f06e1edac">

**Screenshot from Figma:**
<img width="287" alt="image" src="https://github.com/calblueprint/immigration-justice-project/assets/84427964/03e9bd83-c845-4dfb-bed2-8e4b4ee7c9e1">